tests/backends/storage/test_s3.py

```diff
@@ -0,0 +1,387 @@
+"""Tests for Ralph S3 storage backend"""
```

Comment:
```suggestion
"""Tests for Ralph S3 storage backend."""
```
Even if pydocstyle is activated on the tests, I think it's nice to be compliant to its rules in all the files. What do you think?

Reply:
You're right! It should be compliant now.